Polygon_mesh_processing::parameters::choose_parameter has not been declared

## Issue Details

While trying to use `polygon_mesh_to_polygon_soup` I am getting the following errors:

```
/usr/include/CGAL/Polygon_mesh_processing/polygon_mesh_to_polygon_soup.h:80:21: error: ‘CGAL::Polygon_mesh_processing::parameters::choose_parameter’ has not been declared
   using parameters::choose_parameter;
                     ^~~~~~~~~~~~~~~~
/usr/include/CGAL/Polygon_mesh_processing/polygon_mesh_to_polygon_soup.h:81:21: error: ‘CGAL::Polygon_mesh_processing::parameters::get_parameter’ has not been declared
   using parameters::get_parameter;
```

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): Linux 64bits
* Compiler:
* Release or debug mode: (I only tried to compile in Debug)
* Specific flags used (if any): none
* CGAL version: 5.1.1
* Boost version: 1.66
* Other libraries versions if used (Eigen, TBB, etc.):

--- a/Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/polygon_mesh_to_polygon_soup.h
+++ b/Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/polygon_mesh_to_polygon_soup.h
@@ -17,6 +17,7 @@
 
 #include <CGAL/algorithm.h>
 #include <CGAL/boost/graph/iterator.h>
+#include <CGAL/boost/graph/Named_function_parameters.h>
 #include <CGAL/boost/graph/named_params_helper.h>
 #include <CGAL/Container_helper.h>
 #include <CGAL/Dynamic_property_map.h>

